ui/components/header-footer.js

```diff
@@ -70,6 +70,8 @@ export default function HeaderFooter({ children, showSearch }) {
       <Head>
         <title>OMB Policy Library (Beta)</title>
         <link rel="stylesheet" href="/static/styles.css" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.css" integrity="sha384-8QOKbPtTFvh/lMY0qPVbXj9hDh+v8US0pD//FcoYFst2lCIf0BmT58+Heqj0IGyx" crossOrigin="anonymous" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.js" integrity="sha384-GR8SEkOO1rBN/jnOcQDFcFmwXAevSLx7/Io9Ps1rkxWp983ZIuUGfxivlF/5f5eJ" crossOrigin="anonymous" />
```

Comment:
Is this needed? We're compiling KaTeX into the JS, so I don't think we need this separate, client-only asset.

Reply:
👍